packages/@aws-cdk/aws-stepfunctions/README.md

```diff
@@ -159,17 +159,79 @@ similar to (for example) `inputPath`.
 
 #### Lambda example
 
+[Invoke](https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html) a Lambda function.
+
+You can specify the input to your Lambda function through the `payload` attribute.
+By default, no payload is specified so Step Functions invokes Lambda with the empty
+object `{ }` as input.
+
+The following snippet invokes a Lambda Function with the state input as the payload
+by referencing the `$` path.
+
+```ts
+new sfn.Task(this, 'Invoke with task context', {
+  task: new tasks.RunLambdaTask(myLambda, {
+    payload: sfn.TaskInput.fromDataAt('$'),
+  }),
+});
+```
+
+When a function is invoked, the Lambda service sends back the following
+[response elements](https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_ResponseElements)
+
+⚠️ The response from the Lambda function is in an attribute called `Payload`
+
+The following snippet invokes a Lambda Function by referencing the `$.Payload` path
+to reference the output of a Lambda executed before it.
+
+```ts
+new sfn.Task(this, 'Invoke with empty payload', {
+  task: new tasks.RunLambdaTask(myLambda),
+});
+
+new sfn.Task(this, 'Invoke with payload field in the state input', {
+  task: new tasks.RunLambdaTask(myOtherLambda, {
+    payload: sfn.TaskInput.fromDataAt('$.Payload'),
+  }),
+});
+```
+
+The following snippet invokes a Lambda and sets the task output to only include
```

Comment:
"only include" - as opposed to what else?

Reply:
as opposed to all of the response elements `StatusCode`, `ExecutionResult`, etc... https://docs.aws.amazon.com/lambda/latest/dg/API_Invoke.html#API_Invoke_ResponseElements

so this only returns the Lambda return which will be in the `Payload` attribute. I think this is covered in the blurb above though?